compiler/src/iree/compiler/Codegen/VMVX/LowerLinalgMicrokernels.cpp

```diff
@@ -28,6 +29,22 @@ namespace iree_compiler {
 
 namespace {
 
+static llvm::cl::opt<bool> clAssumeDynamicDimsContiguous(
+    "iree-vmvx-microkernels-assume-dynamic-dims-contiguous",
+    llvm::cl::desc(
+        "In the VMVX backend, when deciding whether an op can be rewritten as "
+        "a microkernel call, we need to check if memref layouts inner "
+        "dimeensions are contiguous row-major as expected by the microkernel. "
```

Comment:
dimensions*

Reply:
What do you mean -- it's easy enough to construct a memref with non-contiguous inner dims, just take a memref.subview out of a larger memref. Here the memrefs we are interested in are subviews generated by the tile-and-distribute-to-workgroups pass. That they happen to be contiguous depends on the actual runtime values of SSA values controlling their layout. Issue #11633 referenced in the PR description has details, including the typical post-tiling IR in question in https://gist.github.com/bjacob/59f3ea6bf40aef7915fff9f6a287a599

Reply:
Talked with @MaheshRavishankar . What I was not getting, was that bufferization always creates contiguous buffers.  Therefore the existing contiguousness test was always futile as far as e2e workloads were concerned. We agreed that I would remove the commandline flag, and instead make it a pass option (not exposed on the compiler command line, for debugging only) and default to true.